docs/asyncapi.yaml

```diff
@@ -641,6 +679,7 @@ components:
       example: 80001
       enum:
         - 80001 (Arable and horticultural soils Introductory)
+        - 80299 (SFI Accelerated Payment)
     SchemeId:
```

Comment:
Should be enum

Reply:
SchemeCode:
  type: string
  description: Unique funding option code
  example: 80001
  enum:
    - 80001 (Arable and horticultural soils Introductory)
    - 80299 (SFI Accelerated Payment)
    
    SchemeCode is enum using the code and descriptions provided